resdk/tests/functional/analysis/e2e_differential_expressions.py

```diff
@@ -0,0 +1,35 @@
+# pylint: disable=missing-docstring
```

Comment:
Write some more tests that covers edge cases. I.e. what happens if some samples from the collections are not in any relation,...